homeassistant/components/recorder/statistics.py

```diff
@@ -2057,6 +2087,7 @@ def _generate_statistics_at_time_stmt(
 
 
 def _statistics_at_time(
+    hass: HomeAssistant,
```

Comment:
It's a bit odd we pass around `hass` instead of the recorder instance inside recorder code. But we can improve that in a separate PR.